lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * dummylayout.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+var graph;
+
+function buildGraph(nodes_and_connections) {
```

Comment:
Also, don't bother with a parameter if you're not going to use it. JS will not complain about the mismatched method signature.